doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -1106,13 +1107,21 @@ Response:
       + Members
           + keyValues - when used, the request payload uses the `keyValues` simplified entity
             representation. See "Simplified Entity Representation" section for details.
+          + upsert - when used, entity is updated if already exits. If upsert is not used and
```

Comment:
use backtick for `422 Unprocessable Entity`

Reply:
Fixed in https://github.com/telefonicaid/fiware-orion/pull/3217/commits/f01b3c3472c92bb5a2578e3892737a30ab2fbf5f